packages/atomic/src/components/insight/atomic-insight-interface/e2e/atomic-insight-interface.e2e.ts

```diff
@@ -41,15 +41,18 @@ test.describe('Atomic Insight Panel', () => {
   test.describe('result actions interactions', () => {
     test.beforeEach(async ({insightInterface}) => {
       // Video results have better actions to test
+      const insightSearchPromise =
+        insightInterface.waitForInsightSearchResponse();
       await insightInterface.getTabByName('Videos').click();
-      await insightInterface.waitForNonVideoResultsToBeDetached();
+      await insightSearchPromise;
+      await insightInterface.waitForVideoResultLinksToBeVisible();
     });
 
     test('attach to case', async ({insightInterface}) => {
       await expect(insightInterface.getTabByName('Videos')).toHaveAttribute(
         'active'
       );
-      await insightInterface.hoverResultByIndex(0);
+      await insightInterface.hoverResultTitleByIndex(0);
```

Comment:
@alexprudhomme This broke because of folding, the `.hover()` command puts the mouse at the center of the Locator element. But since we we're dealing with folded results, the middle of the element was actually the folded results, and not the top results.